projects/rero/ng-core/src/lib/record/search/record-search.component.ts

```diff
@@ -490,7 +476,52 @@ export class RecordSearchComponent implements OnInit, OnChanges {
   }
 
   /**
-   * Emit new parameters when a change appends.
+   * Search for records.
+   * @param resetPage If page needs to be resetted to 1.
+   * @param emitParameters If parameters have to be emitted in parents.
+   */
+  private _getRecords(resetPage: boolean = true, emitParameters: boolean = true) {
+    this._lastSearchParameters = this.serializeSearchParameters();
```

Comment:
I think that this can be done at the record service using: https://rxjs-dev.firebaseapp.com/api/operators/distinctUntilChanged and `RequestCache` some examples are here: https://angular.io/guide/http. I can imagine a getRecord method with an option `useCache` set true by default. This can be done in an other PR.

Reply:
Yes, I think we could refactor that later.